exercises/rna-transcription/description.md

```diff
@@ -15,3 +15,7 @@ each nucleotide with its complement:
 * `C` -> `G`
 * `T` -> `A`
 * `A` -> `U`
+
+Note that since we are finding the complement, you should use the mapping of pairs above (and not just swap T for U).
```

Comment:
I'm not sure what you actually want to say with this sentence…

It is semantically the same as the sentence in front of the mapping, but pointing to the mapping above, instead of below:

> Given a DNA strand, its transcribed RNA strand is formed by replacing each nucleotide with its complement:

Reply:
Sure, but in the sourced Rosalind exercise the problem is different. It was just to clarify that.

Reply:
Thats a completely different thing and a big bummer!

This means, that either our exercise or the rosalind exercise in incorrect in terms of biology.

We need to find this out, would you mind open a clean ticket in this repository explaining the circumstances?

Reply:
@NobbZ, in the Rosalind exercise, they are converting the coding strand to RNA, but I find that a very strange approach. Typically people consider transcription with reference to the template strand (which is the complement of the coding strand), and this is definitely what most people would think of if asked to write an RNA transcription program without being given further detail. Regardless, I think the current exercise is probably clear enough about how we are doing it, since it gives a transcription table, and neither approach is technically wrong.